lib/matplotlib/widgets.py

```diff
@@ -4199,19 +4199,31 @@ class Lasso(AxesWidget):
         Whether to use blitting for faster drawing (if supported by the
         backend). See the tutorial :ref:`blitting`
         for details.
+    props: dict, optional
+        Lasso line properties. See `.Line2D` for valid
+    properties. Default *props* are::
+
+        {'linestyle' : '-', 'color' : 'black', 'lw' : 2}
+
+    .. versionadded:: 3.9
```

Comment:
```suggestion
        properties. Default *props* are::

            {'linestyle' : '-', 'color' : 'black', 'lw' : 2}

        .. versionadded:: 3.9
```
Just need to fix the indendation. "properties" on the same level as "Lasso line...", the default on one more indentation level and ``.. versionadded..`` on the same as "properties" and "Lasso line...". (Not sure I got it correct in the change above, hard to see...)

Currently it looks like https://output.circle-artifacts.com/output/job/3d3513c9-2b4e-481a-bc54-cced765e1d06/artifacts/0/doc/build/html/api/widgets_api.html#matplotlib.widgets.Lasso

Reply:
And feel free to rebreak the two first lines of the added doc-string with a maximum line length of 88.

Reply:
Blah... Seriously, thanks for all the help. I *think* I got it this time. 👍